app/views/bookmarks/_filters.html.erb

```diff
@@ -131,6 +131,12 @@
       <dd class="submit actions"><%= f.submit ts("Sort and Filter") %></dd>
     </dl>
 
+    <% if @owner = @bookmarkable || @pseud || @user || @collection || @tag %>
```

Comment:
You should be able to use `<% if @owner %>` here, since the variable is set in the controller. Same goes for the bookmark filters.